src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/locations/LocationsResourceIntTest.kt

```diff
@@ -195,6 +202,25 @@ class LocationsResourceIntTest : IntegrationTestBase() {
         }
     }
 
+    @Test
+    fun `Audit clear fails`() {
```

Comment:
Should we also have a simliar test for when the initial populate audit fails?

Reply:
Done